cmd/entrypoint/waiter.go

```diff
@@ -31,11 +32,14 @@ func (rw *realWaiter) setWaitPollingInterval(pollingInterval time.Duration) *rea
 //
 // If a file of the same name with a ".err" extension exists then this Wait
 // will end with a skipError.
-func (rw *realWaiter) Wait(file string, expectContent bool, breakpointOnFailure bool) error {
+func (rw *realWaiter) Wait(ctx context.Context, file string, expectContent bool, breakpointOnFailure bool) error {
 	if file == "" {
 		return nil
 	}
 	for ; ; time.Sleep(rw.waitPollingInterval) {
+		if err := ctx.Err(); err != nil {
```

Comment:
Do we also want to handle `ctx.Done()` ? 🤔 Not sure if that case would happen but…

Reply:
Thank you for the review. Fixed.
Looking at the code again, I think using your suggestion, i.e. `ctx.Done()` is more readable. Updated.

On a side note, I increased the polling interval on the unit tests from 10 to 25 milliseconds, as it seems to reduce flakes significantly (from about 5% to around 0,5% in my tests). If this needs to be reverted, please let me know.

`const testWaitPollingInterval = 25 * time.Millisecond`